### PR TITLE
LPS-133830 Use updated title and fallback to selected item title

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
ci:test:sf

**Comment by p2kmgcl** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/ItemSelector.js [hunk 1]
```diff
@@ -134,6 +134,28 @@ export default function ItemSelector({
 		[label, onItemSelect, selectedItem]
 	);
 
+	const selectedItemTitle = useSelectorCallback(
```
Comment (victorg1991) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/ItemSelector.js:137]: Your crusade is ever lasting :P

Comment (p2kmgcl) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/common/components/ItemSelector.js:137]: :heart:

**Comment by victorg1991** [comment]
ci:forward

**Pull request closed without merging**